tests/conftest.py

```diff
@@ -10,6 +10,23 @@
 PROD_API_KEY = os.environ['PROD_API_KEY']
 
 
+def pytest_sessionstart(session):
+    # this is for local unit testing with google appengine, otherwise you get a
+    # 'No api proxy found for service "urlfetch"' response
+    try:
+        from google.appengine.ext import testbed
+        session.appengine_testbed = testbed.Testbed()
+        session.appengine_testbed.activate()
+        session.appengine_testbed.init_urlfetch_stub()
+    except Exception:
```

Comment:
When would we arrive at this exception? Would we instead want to raise a message of sorts here instead of passing since we'd want the above to run correctly and not fail silently?

Reply:
The exception is for normal operation when _not_ running in a google appengine setup. This essentially mirrors the library selection mechanism in https://github.com/EasyPost/easypost-python/blob/8af4e9d7fa5d6f71d2cc4625ef546bab489d76e2/easypost/__init__.py#L24-L26

Want me to change to except ImportError instead?

Reply:
Ah, if that's the case it may be advantageous to simply add a comment under the except and above the pass explaining how we'd get there. Your point of `not initialize the urlfetch testbed stub since we're not using the appengine sdk.` could be sufficient!